options/locale/locale_en-US.ini

```diff
@@ -2442,7 +2442,7 @@ release.edit_release = Update Release
 release.delete_release = Delete Release
 release.delete_tag = Delete Tag
 release.deletion = Delete Release
-release.deletion_desc = Deleting a release only removes it from Gitea. Git tag, repository contents and history remain unchanged. Continue?
+release.deletion_desc = Deleting a release only removes it from Gitea. The Git tag, your repository's contents and its history will not be affected. Continue?
```

Comment:
```suggestion
release.deletion_desc = Deleting a release only removes it from Gitea. It will not affect the Git tag, the contents of your repository or its history. Continue?
```